civicase.php

```diff
@@ -72,6 +74,19 @@ function civicase_civicrm_tabset($tabsetName, &$tabs, $context) {
   }
 }
 
+/**
+ * Count the number of Cases for current user.
+ */
+function get_case_count() {
```

Comment:
Why don't we use the Case API here and pass the relevant parameters we need.
The original count function uses the Case.getcount API
Also we should exclude deleted cases too as it was excluded in the function used before.

Se here for reference: https://github.com/civicrm/civicrm-core/blob/5.19/CRM/Case/BAO/Case.php#L1859-L1874

Also this function in core shows that Civi does not exclude cases linked to disabled Case types from the count. Are we sure this is what we really want, deviating from the core behaviour? Did you double check with PM?